### PR TITLE
Data dic ssui

### PR DESCRIPTION
**Pull request by kcarra**

### CODE REVIEW

File: apps/andi/lib/andi_web/live/edit_live_view/data_dictionary_field_editor.ex [hunk 1]
```diff
@@ -30,19 +30,19 @@ defmodule AndiWeb.EditLiveView.DataDictionaryFieldEditor do
         </div>
         <div class="data-dictionary-field-editor__selector">
           <%= label(@form, :name, "Selector", class: "label label--required") %>
-          <%= text_input(@form, :selector, id: id <> "_name", class: "data-dictionary-field-editor__selector input", disabled: !is_source_format_xml(@source_format)) %>
+          <%= text_input(@form, :selector, id: id <> "_name", class: "data-dictionary-field-editor__selector input", disabled: !DataDictionaryHelpers.is_source_format_xml(@source_format)) %>
```
Comment (bbrewer-forge) [inline apps/andi/lib/andi_web/live/edit_live_view/data_dictionary_field_editor.ex:33]: I like this clarification

**Review by bbrewer-forge — APPROVED** [review]

File: apps/andi/test/integration/andi_web/live/metadata_form_test.exs [hunk 2]
```diff
@@ -615,19 +615,13 @@ defmodule AndiWeb.MetadataFormTest do
...
       {:ok, dataset} = Datasets.update(smrt_dataset)
 
       assert {:ok, view, html} = live(conn, @url_path <> dataset.id)
 
-      assert [
-               {"", ""},
-               {"bob@example.com", _},
-               {"hello@world.com", _},
-               {"test@test.com", _}
-             ] = get_all_select_options(html, ".metadata-form__dataset-owner")
```
Comment (cameronmarsh) [inline apps/andi/test/integration/andi_web/live/metadata_form_test.exs:630]: why do

Comment (bbrewer-forge) [reply apps/andi/test/integration/andi_web/live/metadata_form_test.exs:630]: Oh, good catch

Comment (kcarra) [reply apps/andi/test/integration/andi_web/live/metadata_form_test.exs:630]: Users were being added as part of the conncase which the tests had no control over. It was breaking certain scenarios so I just wanted to check that the user that I created was in the list of users

**Pull request merged**